### PR TITLE
Create .gitignore

### PR DESCRIPTION
**Pull request by mebreuer**
Adding back .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+dist/
+build/
+functions/lib/
+js_scripts/
+.env*
+config/firebase-gcp.yml
```
Comment (californianseabass) [inline .gitignore]: remove line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+config/firebase-gmail.yml
```
Comment (californianseabass) [inline .gitignore]: remove line

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
+dist/
+build/
+functions/lib/
+js_scripts/
```
Comment (californianseabass) [inline .gitignore]: remove line

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,12 @@
+dist/
+build/
+functions/lib/
```
Comment (californianseabass) [inline .gitignore]: remove line

**Comment by mebreuer** [comment]
Made changes, take a look! @californianseabass

**Pull request merged**